cluster/mesos/docker/deploy-addons.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
Do you have a bug to get rid of this entirely? It's against where add-ons are headed, and definitely doesn't work with any existing add-on upgrade mechanics. It's closer to where add-ons started when I first filed #3579.